br/pkg/restore/split.go

```diff
@@ -185,12 +185,26 @@ SplitRegions:
 	return nil
 }
 
-func (rs *RegionSplitter) hasRegion(ctx context.Context, regionID uint64) (bool, error) {
+func (rs *RegionSplitter) hasHealthyRegion(ctx context.Context, regionID uint64) (bool, error) {
 	regionInfo, err := rs.client.GetRegionByID(ctx, regionID)
 	if err != nil {
 		return false, errors.Trace(err)
 	}
-	return regionInfo != nil, nil
+	// the region hasn't get ready.
+	if regionInfo == nil {
+		return false, nil
+	}
+
+	// check whether the region is healthy and report.
+	for _, peer := range regionInfo.PendingPeers {
+		log.Warn("unhealthy region detected", logutil.Peer(peer), zap.String("type", "pending"))
```

Comment:
What if we have lots of pending peers in cluster. this will print too many unnecessary logs. I suggest to use metrics rather than log.

Reply:
There isn't a prometheus server pulling BR, hence the metric may lose.😢

I guess we can just set this log level to `DEBUG`.

Reply:
`DEBUG` is ok for me